packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -1,37 +1,156 @@
+import contextlib
+import json
 import logging
+import random
+import threading
+from collections import deque
 from subprocess import check_output
 
 import pytest
-
 import requests
-
 import retrying
 
 from pkgpanda.build import load_json
 from test_util.marathon import get_test_app, get_test_app_in_docker
 
 
+TIMEOUT = 500
+
```

Comment:
This test adds some non-trivial log output on top of what was already a pretty decent amount of logging. Could we please add a module level logger?
```
log = logging.getLogger(__name__)
```
And then migrate any statements from logging to use that log. This way we can crank up the verbosity of just this test for easy debugging